.pre-commit-config.yaml

```diff
@@ -15,3 +15,9 @@ repos:
         stages: [commit-msg]
     repo: https://github.com/jorisroovers/gitlint
     rev: v0.18.0
+  - hooks:
```

Comment:
seems like this repo has also `go-unit-tests` and other hooks, worth looking if we can get some value from them

Reply:
Given that our unit tests also try to spin up containers, I wouldn't try to scope that into this change